content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -314,3 +314,77 @@ $ kubectl delete gateway httpbin-gateway
 $ kubectl delete virtualservice httpbin
 $ kubectl delete --ignore-not-found=true -f @samples/httpbin/httpbin.yaml@
 {{< /text >}}
+
+## (Experimental) Configuring network toplogies
+
+Istio provides the ability to control X-Forwarded-For and X-Forwarded-Client-Cert as an experimental feature.
+
+Many users choose to deploy Istio ingress gateways in using various network topologies
+(e.g. behind Cloud Load Balancers, a self-managed Load Balancer or directly expose the
+Istio ingress gateway). As such, these topologies require different ingress gateway configurations for
+correct client attributes like IP/ports/certs to be propagated to the workload running in the cluster.
+
+Configuration of XFF and XFCC headers are configured by using by using `MeshConfig` during Istio
+*installation*. To simplify configuration, create a single YAML file to pass to `istioctl`.
+
+{{< text yaml >}}
+cat <<'EOF' > topology.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      gatewayTopology:
+EOF
+{{< /text >}}
+
+### Configuring X-Forwarded-For Headers
+
+Many applications, both old and new, rely on client attributes, such as X-Forward-For or X-Forwarded-Proto,
+to be forwarded by reverse proxies in a request. However, due to the variety of network
+topologies Istio can be deployed in, it is required to set the number of trusted proxies deployed in front
+of the Istio gateway proxy.
+
+To set the number of trusted proxies add the following to your `topology.yaml` file.
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      gatewayTopology:
+        numTrustedProxies: <VALUE>
+{{< /text >}}
+
+For example, if you have a cloud based LB, a reverse proxy and then the Istio gateway proxy
+then `<VALUE>` would be 3.
```

Comment:
```suggestion
then `<VALUE>` would be 2.
```

Reply:
Note that all the proxies in front need to parse HTTP traffic and append the X-Forwarded-For headers for every hop. If the number of entries in `X-Forwarded-For` header is less than the number of trusted hops configured, Envoy falls back to using the immediate downstream address as the trusted client address.  Please refer to [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for) for understanding how `X-Forwarded-For` headers and trusted client address is determined by Envoy.

Reply:
added